src/containers/Home/soical media/SocialMedia.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './SocialMedia.css';
+import { TwitterTweetEmbed } from 'react-twitter-embed';
+import { motion } from 'framer-motion';
+
+const SocialMedia = () => {
+  const tweets = [
+    { id: '1291554671321067523' },
+    { id: '1292037842722525184' },
+    { id: '1117802268127113216' },
+    { id: '1291981504868950018' },
+    { id: '1289718054159388673' },
+    { id: '1291796278360367107' },
+  ];
+  return (
+    <section className="social-media   lg:h-xxxl md:h-xxl flex flex-col items-center py-8 ">
```

Comment:
tailwind provide us with height, is there a reason you create your own?

Reply:
The reason why I am doing this is that tailwind provides a max height of h-64 which is 16 rem ! if you want to add more you can extend the heights in the taliwnd.config.js